src/Application/VerifierPresenterTrait.php

```diff
@@ -56,8 +56,15 @@ public function getVerifier(): Verifier
 
     /**
      * Ensures that the action method exists.
+     *
+     * @param string $method
+     * @param array  $parameters
+     *
+     * @throws BadRequestException
+     *
+     * @return bool
      */
-    protected function tryCall($method, array $parameters): bool
+    protected function tryCall(string $method, array $parameters): bool
```

Comment:
This typehint will most likely cause a problem because you would redefine a method with a different header which PHP doesn't allow.